Resources/Prototypes/_Goobstation/Entities/Clothing/Back/satchel.yml

```
@@ -7,6 +7,15 @@
   - type: Sprite
     sprite: _Goobstation/Clothing/Back/Satchels/nanorep.rsi
 
+- type: entity
+  parent: [ClothingBackpackSatchel, BaseCentcommContraband]
+  id: ClothingBackpackSatchelBlueshield
+  name: blueshield satchel
+  description: A very protective satchel, smells like energy gun.
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)